src/server/pfs/server/driver.go

```diff
@@ -141,7 +141,7 @@ func newDriver(
 		return nil, err
 	}
 	if env.StorageDiskCacheSize > 0 {
-		diskCache, err := obj.NewLocalClient(filepath.Join(os.TempDir(), "pfs-cache", uuid.NewWithoutDashes()))
```

Comment:
If we're creating `/tmp` do we need to also move these to CacheRoot? I guess we're adding both to the image so it should work anyways, I'm just not sure if I'm missing anything.

Reply:
No, it's definitely not necessary, it just seemed more consistent (and what I thought would fix the issue initially). I can put it back - I don't imagine too many users are concerned with the actual path we use.